### PR TITLE
#162685671 User can view profiles

### PR DESCRIPTION
**Pull request by fahadmak**
#### What does this PR do?
PR enables users should be able to see the list and profiles of existing authors
#### Description of Task to be completed?
A logged in/authenticated user should be able to see the list and profiles of existing authors. 
#### How should this be manually tested?
Before testing run these commands in cmd

`git clone https://github.com/andela/Ah-backend-guardians.git`

`cd Ah-backend-guardians`

`git checkout ft-view-all-profiles-162685671`

`virtualenv env`

`. env/bin/activate`

`pip install -r requirements.txt`

1.  Create a postgres database and a `.env` file in your root directory and add the following configurations.
```
SECRET_KEY=andela_sjdndkjghdf78872

DB_NAME=your_database_name

DB_USER=database_user

DB_PASSWORD=database_password

DB_HOST=127.0.0.1

ALLOWED_HOSTS=.localhost, .herokuapp.com

EMAIL_PORT=587

EMAIL_HOST_USER = gauthorshaven@gmail.com

EMAIL_HOST_PASSWORD = guardiansauthors
```
2. Open Postman and type `http://localhost:8000/api/profiles/` in url text box. Select GET in methods drop down then click `SEND` button
#### What are the relevant pivotal tracker stories?
[#162685671](https://www.pivotaltracker.com/story/show/162685671)
#### Screenshots (if appropriate)
screen shot 2019-01-21 at 2 44 57 pm

### CODE REVIEW

**Review by VivianDoreen — APPROVED** [review]
LGTM

File: authors/apps/profiles/tests/test_views.py [hunk 1]
```diff
@@ -61,3 +61,16 @@ def test_unauthorized_to_edit_profile(self):
...
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+
+class ListProfileTestCase(BaseTestCase):
+    def setUp(self):
+        username1 = user1['username']
+        username2 = user2['username']
+        self.user = User.objects.get(username=username2)
+
+    def test_get_all_user_profiles(self):
```
Comment (BarnaTB) [inline authors/apps/profiles/tests/test_views.py:70]: Please a docstring comment for this test.

File: authors/apps/profiles/views.py [hunk 2]
```diff
@@ -58,3 +58,10 @@ def get_object(self):
                 'error':  exception_messages.get
                 ('edit_profile_not_permitted').format(username)
             })
+
+
+class ListProfileView(generics.ListAPIView):
```
Comment (BarnaTB) [inline authors/apps/profiles/views.py:63]: Please add a docstring for this class.

**Review by BarnaTB — CHANGES_REQUESTED** [review]
Great work @fahadmak! Just add those docstrings as in the comments and all should be well.

**Review by BarnaTB — APPROVED** [review]
LGTM!

**Review by archbishop12 — APPROVED** [review]
@fahadmak I am merging this.

**Pull request merged**